nucliadb_client/setup.py

```diff
@@ -42,11 +42,5 @@ def load_reqs(filename):
     zip_safe=True,
     include_package_data=True,
     packages=find_packages(),
-    install_requires=requirements,
-    entry_points={
-        "console_scripts": [
-            "nucliadb_export = nucliadb_client.export:run",
-            "nucliadb_import = nucliadb_client.importing:run",
-        ]
-    },
+    install_requires=["nucliadb_sdk"],
```

Comment:
@vangheem what do you think about this?
installing from pip will effectively install nucliadb_sdk.
Maybe it's too much?

Reply:
Yes, too much. I don't see the point here either. No one is using this except us internally.